src/app/views/query-response/graph-toolkit/GraphToolkit.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { connect } from 'react-redux';
+
+import { Label, MessageBar, MessageBarType } from 'office-ui-fabric-react';
+import templates from '../../../../graph-toolkit-examples';
+import { IQuery } from '../../../../types/query-runner';
+import { parseSampleUrl } from '../../../utils/sample-url-generation';
+
+class GraphToolkit extends Component<any> {
```

Comment:
>any [](start = 37, length = 3)

we're using a lot of any which is missing a lot of value of using typescript.

Reply:
Understandable...
Though the reason we did it like this is that we have only one property. It felt like overkill creating a type for it.